backend/api/models/follower.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+import uuid
+
+# Related Model
+from api.models.author import Author
+
+class Follower(models.Model):
+    # Unique ID of this follower Record
+    follow_id  = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False, unique=True)
+    
+    # The person who want to follow
+    follower   = models.ForeignKey(Author, null=False, on_delete=models.CASCADE, related_name="follower")
+
+    # The person who is being followed( The person who should get the request )
+    followee   = models.ForeignKey(Author, null=False, on_delete=models.CASCADE, related_name="followee")
+    
+    # Not sure if needed but added it just in case
+    accepted   = models.BooleanField(default=False)
+
+    url        = models.URLField()
+
+    # NOTE: Not exactly sure how connection to other servers will work but would we need to store author urls?
```

Comment:
Since author id is unique per server, if we want to hold remote authors info, it might need a separate table. Or we can do is to ask info through URL when we need it.

Reply:
U think this is a good possibility, but I would like to discuss that with you all in a meeting tbh, so I will hold off on te addition of another table